<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewReservationDialog</class>
 <widget class="QDialog" name="NewReservationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Nowa rezerwacja</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="batchTab">
      <attribute name="title">
       <string>Turnus</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <item>
          <widget class="QGroupBox" name="mainGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Zamawiający</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="4" column="0" colspan="2">
             <widget class="QLineEdit" name="emailLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>e-mail</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="surnameLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Nazwisko</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="cityLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Miasto</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="postalCodeLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Kod pocztowy</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="countryLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Kraj</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="nameLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Imię</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="numberLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Numer</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="phoneLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Telefon</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="streetLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Ulica</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLineEdit" name="disabilityLineEdit">
              <property name="minimumSize">
               <size>
                <width>147</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Niepełnosprawność</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="roomGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Pokój</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QListView" name="roomListView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="participantGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Uczestnicy</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTableView" name="participantTableView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="additionalGroupBox">
           <property name="title">
            <string>Dodatkowe</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="parkingCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Parking</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="emptyPlaceCheckBox">
              <property name="text">
               <string>Licz puste</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="dateGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Termin</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="beginLabel">
              <property name="text">
               <string>Data przyjazdu:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCalendarWidget" name="beginCalendarWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="gridVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="endLabel">
              <property name="text">
               <string>Data odjazdu:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCalendarWidget" name="endCalendarWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="gridVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="summaryGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Podsumowanie</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="daysLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Ilość dni:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="daysValueLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="roomsLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Ilość pokoi:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="roomsValueLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="participantsLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Ilość osób:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="participantsValueLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="priceLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Koszt:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="priceValueLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="3">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2" rowspan="4">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>streetLineEdit</tabstop>
  <tabstop>numberLineEdit</tabstop>
  <tabstop>postalCodeLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>countryLineEdit</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>disabilityLineEdit</tabstop>
  <tabstop>roomListView</tabstop>
  <tabstop>participantTableView</tabstop>
  <tabstop>parkingCheckBox</tabstop>
  <tabstop>beginCalendarWidget</tabstop>
  <tabstop>endCalendarWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
